website/docs/help-and-support/troubleshooting-guide/query-errors.md

```diff
@@ -9,15 +9,19 @@ The section illustrates common query errors and how to resolve them on Appsmith.
 You may see below errors when working with [API](/core-concepts/connecting-to-data-sources/authentication/connect-to-apis) or [Query](/core-concepts/data-access-and-binding/querying-a-database/) responses.
 
 ### Execution failed with status 5009
-You could see the query/API execution fails and generates an error response. 
+You could see the query/API execution fails and generates an error response:
 
-> **[<QUERY_OR_API_NAME>] action returned an error response. Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched.**
+<Message
+ messageContainerClassName="error"
+messageContent="<QUERY_OR_API_NAME> action returned an error response. Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched."></Message>
 
 
 ![Response larger than the supported size](/img/Query-errors-response-size-larger-than-5MB.png)
 
 #### Error message
-Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched.
+<Message
+ messageContainerClassName='error'
+messageContent='Response size exceeded the maximum supported size of <SIZE_SPECIFIED_IN_FILE> MB. Please use LIMIT to reduce the amount of data fetched.'></Message>
 
 ![Response larger than the supported size error shown in errors tab](/img/Query-errors-response-size-larger-than-5MB-errors-tab.png)
```

Comment:
Let's add the screenshot in Cause section

Reply:
@jnikhila moved both!